lib/abr/simple_abr_manager.js

```diff
@@ -44,8 +44,14 @@ shaka.abr.SimpleAbrManager = class {
 
     /** @private {shaka.abr.EwmaBandwidthEstimator} */
     this.bandwidthEstimator_ = new shaka.abr.EwmaBandwidthEstimator();
-    // TODO: Consider using NetworkInformation's change event to throw out an
-    // old estimate based on changing network types, such as wifi => 3g.
+
+    // Some browsers implement the Network Information API, which allows
+    // retrieving information about a user's network connection.
```

Comment:
This is true, but a comment about the specific change here would be helpful.  Something about why we listen for this event and what we do with it.

Reply:
I added more info